package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "expo-app-auth",
+  "version": "11.1.0",
+  "description": "Expo Unimodule for interfacing with the OpenID library AppAuth",
+  "main": "build/AppAuth.js",
+  "types": "build/AppAuth.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "lint": "expo-module lint",
+    "test": "expo-module test",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module"
+  },
+  "keywords": [
+    "react-native",
+    "expo",
+    "app-auth",
+    "oauth",
+    "openid",
+    "authentication",
+    "auth",
+    "token",
+    "ios",
+    "android"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo-app-auth.git"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo-app-auth/issues"
+  },
+  "contributors": [
+    "Evan Bacon <bacon@expo.io> (https://github.com/evanbacon)"
+  ],
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://docs.expo.dev/versions/latest/sdk/app-auth/",
```

```suggestion
  "homepage": "https://github.com/expo/expo-app-auth",
```
Since the docs will be phased out eventually